diff --git a/sys/conf/kern.pre.mk b/sys/conf/kern.pre.mk
index e13af7cea00..f86fae0607a 100644
--- a/sys/conf/kern.pre.mk
+++ b/sys/conf/kern.pre.mk
@@ -306,6 +306,21 @@ MLXFW_C=	${OFED_C_NOIMP} \
 		-I${SRCTOP}/sys/contrib/xz-embedded/linux/lib/xz \
 		${.IMPSRC}
 
+# DRM C flags.
+DRM_INCLUDES+=	-I${S}/dev/drm/core/include
+DRM_INCLUDES+=	-I${S}/dev/drm/core/include/uapi
+DRM_INCLUDES+=	-I${S}/dev/drm/drmkpi/include/
+
+DRM_CFLAGS=	${CFLAGS} ${DRM_INCLUDES}
+DRM_CFLAGS+=	-include ${S}/dev/drm/drmkpi/include/drm/drm_os_freebsd.h
+DRM_CFLAGS+=	'-DKBUILD_MODNAME="DRMv5.6"'
+DRM_CFLAGS+=	-Wno-cast-qual -Wno-pointer-arith -Wno-missing-prototypes
+# Complain about unsigned long long versus uint64_t, remove -Wformat for now
+.if ${MACHINE_CPUARCH} == "aarch64"
+DRM_CFLAGS+=	-Wno-format
+.endif
+DRM_C=		${CC} -c ${DRM_CFLAGS} ${WERROR} ${PROF} ${.IMPSRC}
+
 GEN_CFILES= $S/$M/$M/genassym.c ${MFILES:T:S/.m$/.c/}
 SYSTEM_CFILES= config.c env.c hints.c vnode_if.c
 SYSTEM_DEP= Makefile ${SYSTEM_OBJS}
diff --git a/sys/conf/options b/sys/conf/options
index 9df829867b6..50a86909d9d 100644
--- a/sys/conf/options
+++ b/sys/conf/options
@@ -95,6 +95,7 @@ COMPAT_CLOUDABI32	opt_dontuse.h
 COMPAT_CLOUDABI64	opt_dontuse.h
 COMPAT_LINUXKPI	opt_dontuse.h
 _COMPAT_LINUX32	opt_compat.h	# XXX: make sure opt_compat.h exists
+COMPAT_DRMKPI	opt_global.h
 COMPILING_LINT	opt_global.h
 CY_PCI_FASTINTR
 DEADLKRES	opt_watchdog.h
