diff --git a/sys/arm64/conf/GENERIC-DRM b/sys/arm64/conf/GENERIC-DRM
new file mode 100644
index 00000000000..db1fffb1729
--- /dev/null
+++ b/sys/arm64/conf/GENERIC-DRM
@@ -0,0 +1,21 @@
+#
+# GENERIC-DRM
+#
+# Custom kernel for arm64 plus DRM
+#
+# $FreeBSD$
+
+#NO_UNIVERSE
+
+include		GENERIC
+ident		GENERIC-DRM
+
+# DRM Support
+options		COMPAT_DRMKPI
+options		COMPAT_LINUXKPI
+
+device		drm
+device		aw_de2_drm
+device		dw_hdmi
+files		"../../dev/drm/extra_patches/files"
+files		"../../dev/drm/extra_patches/files.aw"
