From e8864c3d7fe7d0ff52423b5f07c28d08b7c73312 Mon Sep 17 00:00:00 2001
From: Emmanuel Vadot <manu@FreeBSD.org>
Date: Wed, 2 Jun 2021 10:12:57 +0200
Subject: [PATCH] arm: drm: Add bridges to GENERIC-DRM
Content-Type: text/plain; charset=UTF-8

---
 sys/arm/conf/GENERIC-DRM | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sys/arm/conf/GENERIC-DRM b/sys/arm/conf/GENERIC-DRM
index df1fe081f26..658d5180fac 100644
--- a/sys/arm/conf/GENERIC-DRM
+++ b/sys/arm/conf/GENERIC-DRM
@@ -19,3 +19,4 @@ device		aw_de2_drm
 device		dw_hdmi
 files		"../../dev/drm/extra_patches/files"
 files		"../../dev/drm/extra_patches/files.aw"
+files		"../../dev/drm/extra_patches/files.bridges"
-- 
2.31.1

